.evergreen/config.yml

```diff
@@ -1087,13 +1087,14 @@ buildvariants:
   tasks:
   - name: debian-package-build
 - name: macos
-  display_name: macOS m1 (Apple LLVM)
+  display_name: macOS (arm64) 11.0
```

Comment:
Let’s not change the name here

Reply:
If you feel strongly I'd suggest opening a MONGOCRYPT ticket for the rename but either way let's not do it in this PR since it affects other teams.